components/x-gift-article/src/v2/UrlSection.jsx

```diff
@@ -29,27 +30,30 @@ export const UrlSection = (props) => {
 	}
 
 	return (
-		<div
-			className="o-forms-input o-forms-input--text o-forms-input--suffix js-gift-article__url-section"
-			data-section-id={shareType + 'Link'}
-			data-trackable={shareType + 'Link'}
-		>
-			<input id="share-link" type="text" name={urlType} value={url} readOnly />
-			<button
-				className={`o-buttons o-buttons--big o-buttons--primary ${
-					enterpriseEnabled ? 'o-buttons--professional' : ''
-				}`}
-				aria-label="Copy link of the gift article to your clipboard"
-				onClick={copyLinkHandler}
+		<div>
+			<div
+				className="o-forms-input o-forms-input--text o-forms-input--suffix js-gift-article__url-section"
+				data-section-id={shareType + 'Link'}
+				data-trackable={shareType + 'Link'}
 			>
-				Copy Link
-			</button>
-			{showCopyConfirmation && (
-				<CopyConfirmation
-					hideCopyConfirmation={actions.hideCopyConfirmation}
-					isArticleSharingUxUpdates={isArticleSharingUxUpdates}
-				/>
-			)}
+				<input id="share-link" type="text" name={urlType} value={url} readOnly />
+				<button
+					className={`o-buttons o-buttons--big o-buttons--primary ${
+						enterpriseEnabled ? 'o-buttons--professional' : ''
+					}`}
+					aria-label="Copy link of the gift article to your clipboard"
+					onClick={copyLinkHandler}
+				>
+					Copy Link
+				</button>
+				{showCopyConfirmation && (
+					<CopyConfirmation
+						hideCopyConfirmation={actions.hideCopyConfirmation}
+						isArticleSharingUxUpdates={isArticleSharingUxUpdates}
+					/>
+				)}
+			</div>
+			{props.showMobileShareLinks && <SocialShareButtons {...props} />}
```

Comment:
**question:** do we need the `props.showMobileShareLinks` condition now? It used to be that we would only show share icons at certain breakpoints, but now we show them at all widths so I don't think we need to have a condition for it any more?

Reply:
probably not, happy to remove.

Reply:
You can also remove it from [next-article](https://github.com/Financial-Times/next-article/blob/f4b70e4f39338f79cea091a9fd3a7f1ec89c5b82/client/components/gift-article/index.jsx#L54) when you get to that

Reply:
done [here](https://github.com/Financial-Times/x-dash/pull/722/commits/5c70a7a970a388b8f4bfb67312f812631195c4c6)

Reply:
There are a few other places where that prop is used (I have found two by searching the code for `showMobileShareLinks`) that you haven't deleted...

Reply:
all references to `showMobileShareLinks` should now be gone